src/GlobalScreen/Scope/MetaBar/Factory/hasTitle.php

```diff
@@ -6,8 +6,9 @@
  */
 interface hasTitle extends isItem
 {
-    public function withTitle(string $title) : hasTitle;
-    
+    public function withTitle(string $title, ?string $title_language = null) : hasTitle;
```

Comment:
I'm not sure: do all items `hasTitle` need the $title_langauge? if so, then define the new methods here, if not a new Interface would be helpful

Reply:
additionally, I think there is an Interface `hasAction` as well. the same question there.